content/en/core/process/workflow.md

```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
  - Remove the `Needs Triage` label once triage is complete
  - Document the reasoning by commenting in the issue. This will help reduce mistakes, as the reasoning will be available for everyone to read, and any mistakes there can be rectified.
  
-# Workflow for contributing to cht-docs
-Unlike the above development workflow, the workflow for contributing to [cht-docs](https://github.com/medic/cht-docs) does not involve release management and Acceptance Testing. In order to streamline the process of contributing to docs while still maintaining quality, please submit a pull request with any changes to [cht-docs](https://github.com/medic/cht-docs). One of the repository's maintainers will review the PR, request additional changes as needed, and merge the PR when it is ready. 
+# Documentation Workflow
 
-Where possible, please follow our [docs style guide]({{< ref "docs-style-guide" >}}).
+This documentation site (and related [GitHub site](https://github.com/medic/medic.github.io)) is the CHT’s main resource for informing users on the product. It is designed to provide Community Health Toolkit users clear information on product functionality, as well as the necessary guidelines to accomplish key steps for setting up, deploying, and maintaining the product. 
 
+Documentation is a collaborative effort among all Community members. To improve information on the documentation site, we encourage commenting, asking questions, and sharing ideas for improvement.
 
+## Types of Documentation
+
+Documentation generally falls into three categories:
+
+### Feature Education
+Mainly prepared for a non-technical or semi-technical audience, it focuses on the [core features of the CHT]({{< ref "apps/features" >}}). Feature education includes summaries of feature functionality and screenshots with annotations or explanations of specific parts of the workflow as needed for added clarity. 
+
+### Technical Documentation
+Mainly prepared for a technical audience, it describes the [CHT Core Framework]({{< ref "core" >}}) and its [reference components]({{< ref "apps/reference" >}}). It includes information on technical implementation, design decisions, architecture descriptions, schemas, and source code. 
+
+### How-to Guides and Tutorials
+Mainly prepared for implementing partner capacity building, it includes best practices [quick guides]({{< ref "apps/guides" >}}) and [tutorials]({{< ref "apps/tutorials" >}}) on functional implementation of a feature (or group of features) and system configuration to assist users in completing specific objectives. 
+
+## Issues
+
+All documentation and technical documentation site issues are managed in [Github cht-docs](https://github.com/medic/cht-docs/issues). When creating issues, add the appropriate [labels](https://github.com/medic/cht-docs/labels) to help provide guidance to other community members.
```

Comment:
Are there minimum required labels? Also, just a note that not all GH users will have access to adding labels. As part of updates to this workflow we should consider what doc site issue templates we want, and start fleshing those out.

Reply:
changed this section substantially